modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -61,31 +70,162 @@ This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
-You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
+The kernel options from the distribution and the profile are combined.
```

Comment:
This feels like it's sitting out here on its own, does it need to be attached to a para somewhere?

Reply:
I think the notes should be made part of normal text.